opentech/static_src/src/app/src/redux/reducers/notes.js

```diff
@@ -0,0 +1,68 @@
+import { combineReducers } from 'redux';
+
+import {
+    UPDATE_NOTE,
+    UPDATE_NOTES,
+    START_FETCHING_NOTES,
+    FAIL_FETCHING_NOTES,
+} from '@actions/notes';
+
+function notesFetching(state = false, action) {
```

Comment:
We may want this on the note object or keyed by ID to the note.

I'm thinking of the case where we are fetching already and then the user changes the application they are looking at.

Reply:
But they are submission specific, that's an issue. I guess they'd need to go to the submissions reducer.

Reply:
As mentioned, I might need to look at this area when i am polling for new notes, so lets leave as is for now.